lib/quadquizaminos_web/live/contests/components/contest_component.ex

```diff
@@ -19,19 +19,35 @@ defmodule QuadquizaminosWeb.ContestsLive.ContestComponent do
   @impl true
   def render(assigns) do
     ~L"""
-    <td><%= @contest.name%></td>
+    <div class="hidden table-cell md:p-4"><%= @contest.name%></div>
```

Comment:
![image](https://user-images.githubusercontent.com/43263401/150213250-65978407-2d84-42c8-b50c-267780c4e8ae.png)

From the screen I cannot see the contest that should be under the future and past section yet they exist.  This is because of the hidden property added . I may assume you don't want the table display in mobile layout but to be displayed in medium up screens.

```suggestion
    <div class="hidden md:table-row">
        <%= live_component @socket, QuadquizaminosWeb.ContestsLive.ContestComponent, id: contest.id, current_user: @current_user %>
          </div>
```
```suggestion
    <div class="md:table-cell md:p-4"><%= @contest.name%></div><div class="md:table-cell md:p-4"><%= @contest.name%></div>
    <%= if @current_user.admin? do %>
      <div class="md:table-cell md:p-4"> <%= start_or_pause_button(assigns,@contest) %> </div>
      <div class="md:table-cell md:p-4">
      <button class="<%= maybe_disable_button(@contest) %> <%= if contest_running?(@contest), do: "red" %> icon-button" phx-click="stop" phx-value-contest='<%= @contest.name %>' <%= maybe_disable_button(@contest) %>><i class="fas fa-stop-circle fa-2x"></i></button>
      </div>
    <% end %>

    <div class="md:table-cell md:p-4"><%= timer_or_final_result(assigns, @contest) %> </div>
    <div class="md:table-cell md:p-4"><%= contest_date(assigns, @contest)%> </div>
    <div class="md:table-cell md:p-4"><%= truncate_date(@contest.start_time) %></div>
    <div class="md:table-cell md:p-4"><%= truncate_date(@contest.end_time) %></div>
    <div class="md:table-cell md:p-4"><%= rsvp_or_results_button(assigns, @contest) %></div>
```